### ISSUE TITLE
ukuu installation uninstalled the nvidia drivers

### ISSUE THREAD

**Issue by KeithMyers** [teejee2008/ukuu#119]
Was running Ubuntu 18.04 with kernel 4.15.0-43-generic with Nvidia ppa driver 410.78.

Added the ukuu repository.  Briefly launched the app to see what it looked like.  Not prepared to update anything yet till I have a backup.  Rebooted.  Was met with a ukuu notification that kernel 4.19.0-11 was available and I dismissed the notification window.
Ran a geekbench benchmark and was surprised to see a notification that no CUDA devices were available. ? ? ? ?

Ran nvidia-settings and no driver found.  Ran clinfo and no platforms found.  Nvidia drivers not loaded by the 4.14.0-43 kernel apparently.  ukuu somehow unloaded the nvidia drivers.

**Comment by gothicVI** [comment]
That does not sound likely. Anyhow it is possible to not have drivers on one kernel (4.15.0-43) that are present on another (4.15.0-43).
Just reinstall them and ideally provide installation logs.

**Comment by KeithMyers** [comment]
Yes the drivers were all installed.  All the bits in the correct places and the kernels compiled with the drivers. Tried reinstalling them and that didn't work either. Don't have any logs as the first thing I did was backtrack everything in reverse that had been done to install ukuu. Removed the program and then the repository.  Fought the problem for over four hours trying everything.
Finally resolved with dpkg --configure -a and some other stuff.  Reloaded the ppa drivers from the command line and got back to where I was before I installed ukuu.  I won't attempt the program again until I can figure out how to make a Live USB installation with the Nvidia drivers installed so I can make a partition backup.  That is the main issue now.  Can't boot the stock nouveau drivers with a RTX 2080 card.
